ceph.spec.in

```diff
@@ -2583,7 +2583,6 @@ exit 0
 %config %{_sysconfdir}/prometheus/ceph/ceph_default_alerts.yml
 
 %files mib
-%attr(0755,root,root) %dir %{_datadir}/snmp
-%config %{_datadir}/snmp/mibs
+%{_datadir}/snmp/mibs/*
```

Comment:
Going on the assumption that this directory is owned by `net-snmp` package, as mentioned in https://github.com/ceph/ceph/pull/47115#discussion_r922285525, why `*` if we only provide `CEPH-MIB.txt`?

Reply:
please note, the wildcard is for matching the files under the stuff under the `%{buildroot}`. and i don't want to repeat the file name(s) if possible. so just put `*`.

Reply:
Ah, I missed that.  Just for my education, `%{_datadir}/snmp/mibs` would have also worked, right?  It would have taken all files in `mibs` and all subdirectories (if there were any), whereas `%{_datadir}/snmp/mibs/*` behaves like a shell glob and therefore doesn't recurse?

Reply:
@idryomov yeah, that's also my understanding. see also http://ftp.rpm.org/max-rpm/s1-rpm-inside-files-list.html

and see https://github.com/ceph/ceph/blob/3d710cda2f130aad4278a7debf4da503b42f116e/ceph.spec.in#L1811-L1812 and https://fedora.pkgs.org/36/fedora-aarch64/ceph-mgr-dashboard-16.2.7-11.fc36.noarch.rpm.html for an example for `%{_datadir}/snmp/mibs` -like usage.

i can make the change to use `%{_datadir}/snmp/mibs` instead if you think it'd be better. less typing this way =)

Reply:
Yeah, relying on built-in support for directories seems better than using a wildcard to me.  It's also easier to reason about -- you would just be dropping `%config` directive, the path would remain exactly as is right now.

Reply:
@idryomov updated and repushed.